eland/operations.py

```diff
@@ -1250,6 +1250,40 @@ def to_csv(  # type: ignore
         if path_or_buf is None:
             return "".join(result)
 
+    def to_json(  # type: ignore
+        self,
+        query_compiler: "QueryCompiler",
+        show_progress=False,
+        path_or_buf=None,
+        orient=None,
+        lines=False,
+        **kwargs,
+    ):
+        if orient == "records" and lines is True and isinstance(path_or_buf, str):
```

Comment:
Any reason not to handle path objects and file-like objects?

Reply:
With the latest commit it should handle file-like objects and paths as strings. Let me check on how pandas handles the other kinds of path and implement that as well.

I'll be adding some tests too.

In an earlier commit this was a lot easier to handle, but I mistakenly assumed the Pandas 2.0.0 signature of `to_json`. In that version the `mode` parameter was added (similar as `to_csv` provides. Pandas 1.5 unfortunately does not have this yet, complicating things a bit.